### ISSUE TITLE
Bug: Credit line about this project added more than once

### ISSUE THREAD

**Issue by olleolleolle** [skywinder/github-changelog-generator#507]
Generated changelog files get  a "credit line" added at the end.

Such a "credit line" about this project **should only be added once**. Currently, it is always appended, whether needed or not.

Here is what one of those lines look like:

```
\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```

### Desired behavior

- Avoid adding more lines like it if it already exists
- If such a string (or strings, there can have been many) can be found at the end of the generated file, delete it before adding a new one

### Code to work with

Where to find the code that does it: https://github.com/skywinder/github-changelog-generator/blob/d551c3a1987f34f6d1b20e2c35bcdf52b9a827ab/lib/github_changelog_generator/generator/generator_generation.rb#L23

### Example in the wild

https://github.com/lynndylanhurley/devise_token_auth/commit/5be8770e1e43eeeedc2d94e89d24f6e8d902d0b5

**Comment by Lucashuang0802** [comment]
@olleolleolle problem solved. 😃

**Comment by olleolleolle** [comment]
Fixed in #535. Thanks!
